master/Master.hs

```diff
@@ -269,24 +273,6 @@ consistentHashing msg = do
 --  if acks == Nothing then return ()
 --  else sendDecisionToRingWithRetry decision (timeout * 2)
 
-processMessages :: ReaderT (MVar MasterState) IO ()
```

Comment:
Yeah we should - I only got halfway through refactoring "sendResponses" so haven't gotten to renaming it yet.